roles/core/log_client/vars/RedHat_8.yml

```diff
@@ -1,7 +1,8 @@
 ---
-log_server_packages_to_install:
+log_client_packages_to_install:
+  - rsyslog
   - python3-libselinux
   - python3-policycoreutils
-log_server_services_to_start:
+log_client_services_to_start:
   - rsyslog
-log_server_rsyslog_conf_path: /etc/rsyslog.conf
+log_client_rsyslog_conf_path: /etc/rsyslog.conf
```

Comment:
This variable is the same than in vars/main.yml, you can remove it here.

Reply:
fixed too :-)